maestro-orchestra-models/src/main/java/maestro/orchestra/util/Env.kt

```diff
@@ -34,7 +34,7 @@ object Env {
 
     fun MutableMap<String, String>.withInjectedShellEnvVars(): MutableMap<String, String> {
```

Comment:
This does not need to be a MutableMap if the implementation makes a mutable copy:
```
val mutable = this.asMutableMap()
...
```

Reply:
Good one, updated 👍 thanks Berik!